### ISSUE TITLE
custom entries anomaly unknown level

### ISSUE THREAD

**Issue by andres-erbsen** [coq/coq#9521]
#### Description of the problem

```coq
Declare Custom Entry expr.
Notation "expr0:( s )" := s (s custom expr at level 0).
Check expr0:(_).
(* > Check expr0:(_). *)
(* >              ^ *)
(* Error: Anomaly "Uncaught exception Failure("unknown level 0")." *)
(* Please report at http://coq.inria.fr/bugs/. *)
```

#### Coq Version

8.9.0

**Comment by JasonGross** [comment]
cc @herbelin

**Comment by ejgallego** [comment]
@andres-erbsen do you have permissions to set the tags? If not let us know so we can add you to the proper group.

**Comment by maximedenes** [comment]
@herbelin are you ok with being the assignee?

**Comment by andres-erbsen** [comment]
@ejgallego  I do not have permissions to set tags.

**Comment by ejgallego** [comment]
@maximedenes @Zimmi48 can you folks fix this?

**Comment by Zimmi48** [comment]
I've invited @andres-erbsen.

**Comment by ejgallego** [comment]
Thanks @Zimmi48 !!

**Comment by andres-erbsen** [comment]
thanks @Zimmi48 @ejgallego , accepted

**Comment by staffehn** [comment]
This also applies to the current master.

**Comment by herbelin** [comment]
The fix is easy (sorry for the long delay). I added it to #11530 (which fixes also #9517, #9519, #11331).
